extensions/everhour/src/components/TaskListItem.tsx

```diff
@@ -47,31 +57,30 @@ export function TaskListItem({
   };
 
   const resolveTaskTime = (): string => {
-    const taskTimeToday = timeRecords.find((timeRecord) => timeRecord.id === task.id);
-
-    if (!taskTimeToday) {
-      return "0 min";
+    if (task.time?.recent > 0) {
+      return `${formatSeconds(task.time.recent)} in last 7 days`;
```

Comment:
we can add this for now, but I will potentially adjust this in a following pr just because its very verbose at the moment -- we have some annoyingly long task names and this bloats the lines further atm